cmd/snap/cmd_info_test.go

```diff
@@ -270,8 +272,8 @@ description: |
   https://snapcraft.io/
 snap-id:   mVyGrEwiqSi5PugCwyH7WgpoQLemtTd6
 tracking:  beta
-refreshed: 0001-01-01T00:00:00Z
-installed: 2.10 (1) 1kB disabled
+refreshed: 2006-01-02T22:04:07Z
+current: 2.10 (1) 1kB disabled
```

Comment:
The columns should be aligned in this case, otherwise we still have the same awkward feeling mentioned in the forum, with things next to each other being different for no clear reason.